modules/ossm-control-plane-deploy-1x.adoc

```diff
@@ -108,11 +108,11 @@ $ oc create -n istio-system -f istio-installation.yaml
 $ oc get smcp -n istio-system
 ----
 +
-The installation has finished successfully when the READY column is true.
+The installation has finished successfully when the STATUS column is `InstallSuccessful`.
 +
 ----
-NAME           READY
-basic-install   True
+NAME            READY   STATUS              TEMPLATE   VERSION   AGE
```

Comment:
See my comments on the other PR about the new status format.

Reply:
This is the 1x version of the doc, so I think I'm gonna just leave this the way it is.